docs/source/model_design/risk_factors.rst

```diff
@@ -134,8 +134,131 @@ Definitions
 Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+The **theoretical minimum risk exposure level (TMREL)** is the level of risk
+exposure that would minimize the risk of an adverse outcome for an individual.
+For example, the TMREL for smoking would be "has never smoked." The
+corresponding concept on the population level is the **theoretical minimum risk
+exposure distribution (TMRED)**, which is the distribution of risk exposure that
+would yield the lowest possible population risk. For smoking, the TMRED would be
+the trivial probability distribution assigning everyone in the population to the
+TMREL category "has never smoked." [WHO-Global-Health-Risks-Annex]_,
+[GBD-2017-Risk-Appendix]_
+
+Recall from the `causality section <What is a causal relationship?_>`_ that
+counterfactual analysis is used to describe the causal relationship between a
+risk factor and an outcome. **The TMRED is a particular choice of counterfactual
+exposure distribution** used for the causal attribution of disease burden to a
+given risk factor in a population (see `Population Attributable Fraction
+(PAF)`_). Other choices of counterfactual include the *plausible* minimum risk,
+*feasible* minimum risk, and *cost-effective* minimum risk, each of which can
+obviously depend on specific attributes of the population under consideration.
+On the other hand, Murray et al. state
+[Comparative-quantification-health-risks]_:
+
+  Biological principles as well as considerations of equity would necessitate
+  that, **although the exposure distribution for theoretical minimum risk may
+  depend on age and sex, it should in general be independent of geographical
+  region or population.**
+
+However, they go on to add:
+
+  Exceptions to this are however unavoidable. An example would be the case of
+  alcohol consumption, which in limited quantities and certain patterns, has
+  beneficial effects on cardiovascular mortality, but is always harmful for
```

Comment:
I thought they changed this claim: http://www.healthdata.org/news-release/new-scientific-study-no-safe-level-alcohol